app/views/admin/facilities/upload.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Upload Facilities</h1>
+
+<h3>Instructions:</h3>
+<ol>
+  <li>Download and fill in <%= link_to 'this template', '/documents/upload_facilities.csv'%> to create multiple facilities via file upload.</li>
```

Comment:
Check if you can generate the asset path

Reply:
Tried doing this, but it seems like overkill
Need to:
- Create a route
- Add a method under the facilities controller
- Add a policy for the method

Do you think its worth doing?